apps/windows_explorer/windows_explorer.py

```diff
@@ -1,7 +1,51 @@
+"""
+Windows Explorer Integration for Talon Voice Control
+
+This module provides voice control integration for Windows File Explorer by:
+
+1. LANGUAGE DETECTION & LOCALIZATION:
+   - Detects system UI language (English, German, etc.)
+   - Maps localized folder names to actual filesystem paths
+   - Handles different Explorer window title formats per language
+
+2. WINDOW TITLE PROCESSING:
+   - Strips Explorer-specific suffixes (e.g., "- File Explorer", multi-tab indicators)
+   - Uses regex patterns to clean window titles for path extraction
+   - Supports both single and multi-tab Explorer windows
+
+3. ONEDRIVE INTEGRATION:
+   - Detects OneDrive installation and folder structure
+   - Maps standard folders (Desktop, Documents, Pictures) to OneDrive paths when applicable
+   - Handles hybrid OneDrive/local folder scenarios
+
+4. PATH MAPPING & EXCLUSIONS:
+   - Maps localized display names to actual filesystem paths
+   - Excludes system dialogs and non-navigable windows
+   - Handles user display name mapping for personalized folders
+
+5. VOICE COMMAND ACTIONS:
+   - Navigation (open parent, go to directory)
+   - File operations (new folder, open file, properties)
+   - Address bar control (focus, copy, navigate)
+   - Terminal integration (open command prompt in current location)
+
+This has been tested with:
+   - Windows 11 24H2, Version 10.0.26100 Build 26100, English and German system language
+"""
+
+import logging
 import os
+import re
 
 from talon import Context, Module, actions, app, ui
 
+# Configuration
+DEBUG_LOGGING = False
```

Comment:
In general, we tend to use Talon settings rather than constants in the code to configure logging verbosity.